tests/performance/local/future_overhead.cpp

```diff
@@ -389,6 +411,20 @@ void measure_function_futures_create_thread_hierarchical_placement(
     hpx::lcos::local::latch l(count);
 
     auto sched = hpx::threads::get_self_id_data()->get_scheduler_base();
+
+    if (std::string("core-shared_priority_queue_scheduler") ==
+        sched->get_description())
+    {
+        sched->add_remove_scheduler_mode(
+            hpx::threads::policies::scheduler_mode(
+                hpx::threads::policies::assign_work_thread_parent),
+            hpx::threads::policies::scheduler_mode(
+                hpx::threads::policies::enable_stealing |
+                hpx::threads::policies::enable_stealing_numa |
+                hpx::threads::policies::assign_work_round_robin |
+                hpx::threads::policies::steal_after_local |
+                hpx::threads::policies::steal_high_priority_first));
+    }
```

Comment:
Can we enable these flags unconditionally or will the local/static schedulers complain about unsupported flags?